src/main/webapp/app/shared/markdown-editor/monaco/markdown-editor-monaco.component.ts

```diff
@@ -37,6 +37,9 @@ interface MarkdownActionsByGroup {
     meta: MonacoEditorAction[];
 }
 
+const EXTERNAL_HEIGHT = 'external';
+const BORDER_WIDTH_OFFSET = 3;
```

Comment:
Why 3? -> The editor will often cover the container border if I set it to the exact value (2). This preserves the border at the cost of it sometimes being a little too thick (though the editor scrollbar is probably in the way already)

Reply:
Can you also write this as a comment in the file?

Reply:
Yes